{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py

```diff
@@ -41,81 +36,110 @@
+            'extensions': DEFAULT_EXTENSIONS + [
+                'webpack_loader.contrib.jinja2ext.WebpackExtension',
+                'compressor.contrib.jinja2ext.CompressorExtension',
+                '{{cookiecutter.package_name}}.apps.site.extensions.DjangoNow',
+            ],
+            'bytecode_cache': {
+                'name': 'default',
+                'backend': 'django_jinja.cache.BytecodeCache',
+                'enabled': False,
+            },
+            'autoescape': True,
+            'auto_reload': False,
+            'translation_engine': 'django.utils.translation',
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.contrib.messages.context_processors.messages',
+                'django.template.context_processors.request',
+                'cms.context_processors.settings',
+                'cms.apps.pages.context_processors.pages',
+                'social_django.context_processors.backends',
+                'social_django.context_processors.login_redirect'
+            ]
+        }
+    },
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            os.path.join(SITE_ROOT, 'templates'),
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.contrib.messages.context_processors.messages',
+                'django.template.context_processors.request',
+                'cms.context_processors.settings',
+                'cms.apps.pages.context_processors.pages',
+                'social_django.context_processors.backends',
+                'social_django.context_processors.login_redirect'
+            ]
+        }
+    }
+]
+# Namespace for cache keys, if using a process-shared cache.
+CACHE_MIDDLEWARE_KEY_PREFIX = '{{cookiecutter.package_name}}'
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
+        'LOCATION': '127.0.0.1:11211',
+    }
+}
 
 
-# Locale settings.
+###
+#  ___             _           _                         _   __  _
+# | _ \ _ _  ___  (_) ___  __ | |_    ___ _ __  ___  __ (_) / _|(_) __
+# |  _/| '_|/ _ \ | |/ -_)/ _||  _|  (_-<| '_ \/ -_)/ _|| ||  _|| |/ _|
+# |_|  |_|  \___/_/ |\___|\__| \__|  /__/| .__/\___|\__||_||_|  |_|\__|
+#               |__/                     |_|
+###
```

Comment:
There's a lot more than this which is project-specific; half the stuff in the file will change per-project. But as it goes, the locale + internationalisation settings are something that *isn't* particularly project-specific as these settings are in nearly every one of our projects. This could do with a better title.

Reply:
I wanted the section to be the ones you change post cookie cutter. I've only really changed stuff in the third party section and this section. If we're taking the localisation stuff out I might as well merge it into the base section since it'll only be the apps list and NEWS_APPROVAL_SYSTEM

Reply:
Move the localisation stuff into its own section then figure out where to put `NEWS_APPROVAL_SYSTEM`.